contracts/pm/TicketBroker.sol

```diff
@@ -212,6 +234,12 @@ contract TicketBroker {
         emit UnlockCancelled(_senderAddress);
     }
 
+    // Override
+    function processDeposit(uint256 _amount) internal;
```

Comment:
Is there a scenario where `processDeposit()` and `processPenaltyEscrow()` would have different behavior? What if we just had a single internal function that subtypes should be override that is called by `_fundDeposit()` and `_fundPenaltyEscrow()`?

Reply:
I think you're right. At the moment there is absolutely no difference, so yes I will make the change as you suggested. Thanks!